modules/ingress/variables.tf

```diff
@@ -33,8 +35,8 @@ variable "default_backend" {
     service_port = string
   })
   default = {
-    service_name = "dummy"
-    service_port = "80"
+    service_name = null
```

Comment:
I think we don't need to have default values or those should not be null as not supplying values should not result in strange behaviour.
If those values are mandatory then we could make variables out of those and make them mandatory there.